agent/uiserver/ui_template_data.go

```diff
@@ -31,6 +31,14 @@ func uiTemplateDataFromConfig(cfg *config.RuntimeConfig) (map[string]interface{}
 		uiCfg["metrics_provider_options"] = json.RawMessage(cfg.UIConfig.MetricsProviderOptionsJSON)
 	}
 
+	v2CatalogEnabled := false
+	for _, experiment := range cfg.Experiments {
+		if experiment == "resource-apis" {
```

Comment:
`v1/agent/self` is an interesting endpoint 🤔 and the `DebugConfig` field seems to be completely undocumented on the consul apis page.

Does "enabled" here mean the DC has a V2 service registered or just that it has the permission to register one?

Reply:
My understanding is it's like a feature flag, not telling you that you actually have services registered. However, if you have this experiment enabled, requests to V1 catalog endpoints don't succeed.